src/explore-education-statistics-common/src/components/LinkContainer.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './LinkContainer.module.scss';
+
+interface Props {
+  url: string;
+}
+
+const LinkContainer = ({ url }: Props) => (
```

Comment:
From context and styling, is this something akin to an inline code block? In which case, seems like `LinkContainer` might be too specific of a component.

Reply:
I suspect that we may want to add buttons to copy/open the link down the line, will also be used for fast-track link generation too.